lib/frame_window_filtering/background_subtraction.h

```diff
@@ -25,7 +25,7 @@ class BackgroundSubtraction : public FrameWindowFiltering {
 
 private:
   FrameWindow _cage_frame = FrameWindow();
-  double _threshold = 0.01;
+  double _threshold = 1;
```

Comment:
Would change name to something more descriptive. It's not a threshold it's a factor multiplied by the threshold. "_factor" or something makes more sense.